#An vector of odometry of moving obstacles
Header header
int32 number_of_obstacles
nav_msgs/Odometry[] odom
